### ISSUE TITLE
ChatInterface rendered inside a Block appears very small in size

### ISSUE THREAD

**Issue by yvrjsharma** [gradio-app/gradio#7714]
### Describe the bug

When the ChatInterface is integrated within a Gradio block, the resulting chat window appears to be significantly smaller in size than expected. This can make the chat interface difficult to use and may cause inconvenience to the user.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

While rendering ChatInterface inside a block:

```python
import gradio as gr
def chat(msg, hist):
    return "ok"

ci = gr.ChatInterface(chat)
with gr.Blocks() as demo:
    ci.render()
demo.launch()
```

Implementing ChatInterface directly:

```python
import gradio as gr
def chat(msg, hist):
    return "ok"

ci = gr.ChatInterface(chat)
ci.launch()
```

### Screenshot

While rendering ChatInterface inside a block:

![image](https://github.com/gradio-app/gradio/assets/48665385/05b8e1b7-6a33-4a4b-923f-f880401ffb37)

Implementing ChatInterface directly:

![image](https://github.com/gradio-app/gradio/assets/48665385/310c4f6c-d864-4849-9693-c92e93082a7a)

### System Info

```shell
Testing on Colab
```

### Severity

I can work around it

**Comment by dawoodkhan82** [comment]
@yvrjsharma I think this is expected.

To fix you can set `fill_height=True`

```
import gradio as gr
def chat(msg, hist):
    return "ok"

ci = gr.ChatInterface(chat)
with gr.Blocks(fill_height=True) as demo:
    ci.render()
demo.launch()
```

**Comment by yvrjsharma** [comment]
Ah, this is perfect @dawoodkhan82 ! Thanks for sharing the solution.

**Comment by yvrjsharma** [comment]
Actually, I am still experiencing issues with the chat window size while customizing a ChatInterface using the chatbot parameter.

```
import gradio as gr
def chat(msg, hist):
    return "ok"

chatbot= gr.Chatbot(placeholder='''<img src="https://raw.githubusercontent.com/gradio-app/gradio/main/readme_files/gradio.svg" style="width:30%; opacity:0.5;">''')
ci = gr.ChatInterface(chat, chatbot=chatbot)
with gr.Blocks(fill_height=True) as demo:
    ci.render()
demo.launch()

```

![image](https://github.com/gradio-app/gradio/assets/48665385/817437a1-09e6-4aad-97a9-4adac1cbd2d5)

**Comment by yvrjsharma** [comment]
Hi @dawoodkhan82 I found that `fill_height=True` only works for `gr.ChatInterface`. When using `gr.Chatbot` alone or to customize the ChatInterface, the parameter stops working.

Repro:

- multimodal chatbot with gr.Chatbot (doesn't work): 
```python
import gradio as gr

# multimodal chatbot
def bot_response(history, message):
    for x in message["files"]:
        history.append([(x,), None])
    if message["text"] is not None:
        history.append([message["text"], None])
    response = "**That's cool!**"
    history[-1][1] = ""
    for character in response:
        history[-1][1] += character
        time.sleep(0.05)
        yield history

with gr.Blocks(fill_height=True) as demo:
    chatbot = gr.Chatbot(
        [],
        elem_id="chatbot",
        bubble_full_width=False,
    )

    chat_input = gr.MultimodalTextbox(interactive=True, file_types=["image"], placeholder="Enter message or upload file...", show_label=False)
    chat_input.submit(bot_response, [chatbot, chat_input], [chatbot,])

demo.queue()
if __name__ == "__main__":
    demo.launch(debug=False)	
```
Result:
![image](https://github.com/gradio-app/gradio/assets/48665385/e4f29c50-773c-4165-a601-25b26adf425f)

- Simple Chatbot with gr.Chatbot (doesn't work) : 
```python
import gradio as gr
import random

with gr.Blocks(fill_height=True) as demo:
    chatbot = gr.Chatbot()
    msg = gr.Textbox()
    clear = gr.ClearButton([msg, chatbot])

    def respond(message, chat_history):
        bot_message = random.choice(["How are you?", "I love you", "I'm very hungry"])
        chat_history.append((message, bot_message))
        return "", chat_history

    msg.submit(respond, [msg, chatbot], [msg, chatbot])

if __name__ == "__main__":
    demo.launch()	
```
Result:
![image](https://github.com/gradio-app/gradio/assets/48665385/62d3baa6-32a7-4aaf-9728-6d2c89dbb751)

- ChatInterface with Chatbot customization (doesn't work):
```python
import gradio as gr
def chat(msg, hist):
    return "ok"

chatbot = gr.Chatbot()
with gr.Blocks(fill_height=True) as demo:
  ci = gr.ChatInterface(chat, chatbot=chatbot)
demo.launch()
```
Results: 
![image](https://github.com/gradio-app/gradio/assets/48665385/6edc515f-8fb4-4ab4-9507-9db9c034b69b)

- ChatInterface with Chatbot customization (works):
```python
import gradio as gr
def chat(msg, hist):
    return "ok"

ci = gr.ChatInterface(chat)
with gr.Blocks(fill_height=True) as demo:
    ci.render()
demo.launch()
```
Results:
![image](https://github.com/gradio-app/gradio/assets/48665385/f2f260e8-a28a-4553-b4e8-4f23c6967291)

**Comment by dawoodkhan82** [comment]
@yvrjsharma You can add `scale=1` to the chatbot instance in addition to `fill_height=True` in blocks. That should fix the height issue. Although maybe setting `fill_height=True` should automatically set the scale (like we do in `chat_interface`). @pngwn Thoughts?

**Comment by yvrjsharma** [comment]
Sweet, thanks @dawoodkhan82! I was able to expand all the chatbot test-cases (from above) to full screen height by setting up `scale=1` inside the chatbot instances. I think we should update our [Chatbot documentation](https://www.gradio.app/docs/gradio/chatbot#initialization) to make this clearer, or we could simply use `fill_height=True` to handle this automatically, which would be a cleaner solution.

**Comment by WH-Yoshi** [comment]
Hi, I'm facing this issue too after finding out I'm not applicable for @dawoodkhan82 idea.
Here's my ChatInterface :
![ToThis](https://github.com/user-attachments/assets/5093e226-bf0c-40a8-8a4e-f16863135437)
What I want to create is a `gr.Row()` with the TextBox and the ChatInterface as you can see, but the parameter `scale=1` doesn't exist in `gr.Row()`.
I tried some css and all the `fill_height=True` possible but none will expand te chatbot.
Here's my code :
```python
if __name__ == '__main__':
    CSS = """"""

    chatbot = gr.Chatbot(
        scale=1
    )
    textbox = gr.Textbox(
        lines=2,
        scale=7,
    )

    with gr.Blocks(fill_height=True) as demo:
        with gr.Row(equal_height=True):
            yourself = gr.Textbox(
                placeholder="Tell me more about you and why you are here...",
                scale=2,
            )
            with gr.Column(scale=5):
                gr.ChatInterface(
                    fn=predict,
                    chatbot=chatbot,
                    textbox=textbox,
                    fill_height=True,
                )

    demo.launch()```

Do you guys have any ideas ?

**Comment by abidlabs** [comment]
Closed via #9571
